File: 02_frontend/student_cdv/src/app/components/common/student.ts
export class Student {
  // @ts-ignore
  id: number;
  // @ts-ignore
  name: String;
  // @ts-ignore
  email: String;
  // @ts-ignore
  branch: String;


  constructor(name: String, email: String, branch: String) {
    this.name = name;
    this.email = email;
    this.branch = branch;
  }
}

File: 01_backend/src/main/java/com/cdv/students/entity/Student.java
package com.cdv.students.entity;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Table(name = "student")
@Getter
@Setter
public class Student {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "student_id")
    private Long Id;
    @Column(name = "student_name")
    private String name;
    @Column(name = "student_email")
    private String email;
    @Column(name = "student_branch")
    private String branch;

    public Student() {
    }

    public Student(String name, String email, String branch) {
        this.name = name;
        this.email = email;
        this.branch = branch;
    }
}

